ui/src/components/list-view/QasListView.vue

```diff
@@ -58,6 +59,12 @@ export default {
 
   mixins: [contextMixin, viewMixin],
 
+  provide () {
+    return {
+      isFetchSuccess: computed(() => this.isFetchSuccess)
```

Comment:
pq precisou fazer dessa forma?

Reply:
apesar do data ser reativo, precisei forçar a reatividade do data para que quando atualizasse aqui, atualizasse aonde pego pelo inject

Reply:
- isFetchListSuccssed
- precisamos documentar nossos provides, tem alguma sugestão padrão? que fique naquele arquivo yml.